sphinx/source/docs/user_guide/source_examples/extensions_example_widget.py

```diff
@@ -303,7 +76,7 @@ class JQueryRangeSlider(InputWidget):
     # of JavaScript (or CoffeeScript) code that implements the JavaScript side
     # of the custom extension model.
 
-    __implementation__ = jquery_ui_range_slider_coffee
+    __implementation__ = open('extensions_jquery_ui_range_slider.coffee').read()
```

Comment:
@djpugh this (and the one above) can actually just be the filename  `"extensions_jquery_ui_range_slider.coffee"` Any string ending in `.coffee` will automatically be treated as a file and read in. This is what we should demonstrate.